qiskit/transpiler/_transpiler.py

```diff
@@ -44,7 +45,7 @@ def transpile(circuits, backend, basis_gates=None, coupling_map=None, initial_la
     Raises:
         TranspilerError: in case of bad compile options, e.g. the hpc options.
     """
-    if isinstance(circuits, QuantumCircuit):
+    if not isinstance(circuits, list):
```

Comment:
why?

Reply:
Oh, this was a mistake. I meant to keep this in #911, it's to avoid the circular import on quantum circuit when adding a `qiskit.tools.visualization` import to the `qiskit._quantumcircuit` module.